testutil/testnode/full_node_test.go

```diff
@@ -75,10 +79,31 @@ func (s *IntegrationTestSuite) Test_Liveness() {
 
 func (s *IntegrationTestSuite) Test_PostData() {
 	require := s.Require()
-	_, err := s.cctx.PostData(s.accounts[0], namespace.RandomMessageNamespace(), tmrand.Bytes(100000))
+	_, err := s.cctx.PostData(s.accounts[0], flags.BroadcastBlock, namespace.RandomMessageNamespace(), tmrand.Bytes(100000))
 	require.NoError(err)
 }
 
 func TestIntegrationTestSuite(t *testing.T) {
 	suite.Run(t, new(IntegrationTestSuite))
 }
+
+func (s *IntegrationTestSuite) Test_FillBlock() {
+	require := s.Require()
+
+	for squareSize := 16; squareSize < appconsts.MaxSquareSize; squareSize *= 2 {
```

Comment:
[question] out of curiosity, why start squareSize at 16? My guess is that the estimation logic for non message-shares may break for `squareSizes < 16`.

Reply:
good catch, that was a remnant of the old implementation.

https://github.com/celestiaorg/celestia-app/pull/866/commits/57744bf9c5c3fa6333c96b1e209be3db505dd175

Reply:
also #867